.gitignore

```diff
@@ -5,3 +5,7 @@ plugins/gateway/kong/bin/*
 /bin/*
 *.gob
 goreleaser_artifacts/
+
+# helm
+*.tgz
+Chart.lock
```

Comment:
[helm dependency update](https://helm.sh/docs/helm/helm_dependency_update/) pulls down the upstream dependent charts, so newer updates to upstream charts are not maintained in this git repo. 

The downside is that it adds an additional step ```helm dependency update``` before installing the chart, but keeps the repo clean of binary blobs (there is a bit of trade-off)

I am ok if we leave *.tgz file in the repo, but we have to keep in mind that in the long-term it may bloat the git repo.

Reply:
So @sriaradhyula are you suggesting to keep the Chart.lock and remove the tgz ?

Reply:
> so newer updates to upstream charts are not maintained in this git repo

@sriaradhyula not sure I understand why we need to pull newer updates, we want to stick with a version that we tested and it worked, no?

> The downside is that it adds an additional step helm dependency update before installing the chart

Who will need to run `helm dependency update` the user or us in the workflow before packaging the chart?